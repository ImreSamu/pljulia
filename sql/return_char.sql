CREATE FUNCTION julia_char()
RETURNS CHAR AS $$
    "char"
$$ LANGUAGE pljulia;
SELECT julia_char();
DROP FUNCTION julia_char();
