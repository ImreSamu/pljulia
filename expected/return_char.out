CREATE FUNCTION julia_char()
RETURNS CHAR AS $$
    "char"
$$ LANGUAGE pljulia;
SELECT julia_char();
 julia_char 
------------
 char
(1 row)

DROP FUNCTION julia_char();
